src/utils/walletSearch.js

```diff
@@ -50,12 +51,20 @@ export const wallet = {
   async init (successCallback) {
     // Open iframe with Wallet if run in top window
     window !== window.parent || await this.getReverseWindow();
-    //
+
     this.client = await RpcAepp({
       name: 'AEPP',
-      nodes: [{ name: 'test-net', instance: await Node({ url: NODE_URL, internalUrl: NODE_INTERNAL_URL }) }],
-      compilerUrl: COMPILER_URL
+      nodes: [
+        {name: 'ae_uat', instance: await Node({url: TESTNET_URL})},
+        {name: 'ae_mainnet', instance: await Node({url: MAINNET_URL})}
+      ],
+      compilerUrl: COMPILER_URL,
+      onNetworkChange (params) {
+        this.selectNode(params.networkId); // params.networkId needs to be defined as node in RpcAepp
```

Comment:
how does this work if the wallet tries to switch to a custom network?

Reply:
doesn't work with AEX-2, this is a known issue in the specification